cordova-lib/spec-plugman/platforms/windows.spec.js

```diff
@@ -192,34 +203,39 @@ beforeEach(function () {
 
                 projectsAddedToSpies.forEach(function (spy) {
                     expect(spy).toHaveBeenCalled();
+                    spy.calls.reset();
```

Comment:
Thanks @stevengill ! All comments have been addressed. Let me know if there is anything else!